src/components/appointments/CreateAppointment/AppointmentProgress.js

```diff
@@ -47,26 +50,59 @@ const AppointmentProgress = ({title, currentStep}) => {
     };
   });
 
-  const ProgressIndicatorDisplayComponent =
-    config?.displayComponents?.progressIndicator ?? ProgressIndicatorDisplay;
+  // Add the fixed steps to the the original steps array
+  const stepsToRender = addFixedSteps(
```

Comment:
appointments starts directly at the `producten`, there's no `startpagina` step.

I think re-using this utility is wrong - the step building is similar, but it's not the same so I'd prefer duplicating those step definitions and keeping it local. Imagine that you add another step between overview/confirmation, then the appointments would need to be updated for that too and it's not a guarantee that applies.

Reply:
I didn't realize that. On the test env and in my local forms the start page was there (even in appointments), that's why I added it to this as well. I will remove it.